pages/operations/managing-applications/delete-application.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Delete an Application
+description: How to uninstall your Plural applications.
+---
+
+To uninstall applications run the following command in the CLI or Cloud Shell:
+
+```shell {% showHeader=false %}
+plural destroy <app-name>
+```
+
+This will:
+
+- Destroy all Terraform resources
+- Bring down application in your cloud
+- Empty the Kubernetes namespace associated with your application
+
+This will **not**:
+
+- Remove application builds from your local Plural Git repository
+- Remove application configuration values from `context.yaml`
+
+We don't remove certain information as it contains required configuration for spinning up
+the same instance again. If we removed that information, you would be unable to restore an application after destroying it.
```

Comment:
Is there an option to remove it entirely / wipe it from your Git repo too?

Reply:
Fair question, but there aren't many use cases for this, have you received any feedback from users that this is an important workflow for them?

The short answer I believe is to just manually remove the directory yourself, but it's not recommended

Reply:
All good - we can see if anyone asks in the future. This was the closest ask: 
"hey, so yesterday I did cd bootstrap/terraform && terraform destroy so my infra was cleaned up. However, when I go to plural I see "installed packages", very curious if I can reset all the state in plural."